.travis.yml

```diff
@@ -24,17 +24,8 @@ cache:
     - $HOME/.composer/cache
 
 install:
-  - >
-    composer require-typo3-version "$TYPO3_VERSION";
-  - >
-    echo;
-    echo "Installing the dependencies";
-    composer update --with-dependencies --prefer-stable --prefer-dist ${DEPENDENCIES_PREFERENCE};
-    composer show;
```

Comment:
Looks very good! But please let's keep the `composer show` as this will make debugging problems with dependency versions a lot easier.

Reply:
> keep the composer show

Hm, the output of `composer install` already shows the exact package versions installed. What other benefits does `composer show` provide for debugging?

Reply:
Oh, I had completely missed that `composer install` shows the version. Then this indeed is not needed.